packages/rancher-logging/overlay/templates/loggings/eks/logging.yaml

```diff
@@ -16,10 +16,16 @@ spec:
       Tag: "eks"
       Path: "/var/log/messages"
       Parser: "syslog"
-    {{ with .Values.fluentbit_tolerations }}
     tolerations:
+      - key: "cattle.io/os"
```

Comment:
can we put this with the other tolerations? The more configurable the better for users.

Reply:
In ```values.yaml``` rather than hardcoded here? I'd be fine with that, but I'm curious: if we do not want these pods to run on Windows, should we even expose that customization?

Reply:
We don't want it scheduled on windows nodes but I think it is still important to give the users the option t. I don't have a use case for not having this toleration but putting it in values saves us the pr later if one does come up.

Reply:
Hmmm, ok will do then!